### DISCUSSION TITLE
How to read the "Discover" Feed without login?

### DISCUSSION THREAD

**Discussion by TeteuXD2** [MarshalX/atproto#433] [Questions] [answered]
I am making a python script to read feeds and i put a option to read the discover feed without needing to login, but is just giving me a 401 error

**Comment by MarshalX** [comment]
Hello @TeteuXD2, check this out. It should be the same for you: https://github.com/MarshalX/atproto/discussions/292#discussioncomment-8633413

**Reply by TeteuXD2** [reply]
thanks, but i should replace "test.marshal.dev" with what now?

**Reply by MarshalX** [reply, marked as answer]
> thanks, but i should replace "test.marshal.dev" with what now?

I mean you should create client with public API instead. That's it
